appium/README.md

```diff
@@ -24,10 +25,18 @@ To run a particular test:
 
 To run all tests: `npm run-script test.live.all` or `npm run-script test.mock.all`
 
+## Write and debug tests
+Tips for debugging:
+- For debugging purposes instead of running `bundle exec fastlane build` you can copy `FlowCrypt.app` from `/DerivedData/FlowCrypt-.../Build/Products/Debug-iphonesimulator` (In Xcode open `Products` folder -> `FlowCrypt` -> `Show in Finder`) to appium folder. 
+- Remove contents of `appium/tmp` before test execution. 
+- Execute tests and check `appium/tmp` for troubleshooting.
```

Comment:
In that case `appium/tmp` folder contents should be removed automatically before each appium test, as this is clearly not convenient

Reply:
I thought this is just for troubleshooting?